vcpkg-script/Readme.md

```diff
@@ -0,0 +1,30 @@
+###Instructions to build the third-party dependencies
```

Comment:
Formatting for this file is incorrect. Have a look here:
https://github.com/vcebollada/windows-dependencies/tree/vcpkg/vcpkg-script